### DISCUSSION TITLE
Project naming explanation

### DISCUSSION THREAD

**Discussion by CorruptionVR** [yume-chan/ya-webadb#506] [Q&A] [answered]
why is the project called "Tango"? does it have any symbolical meaning, or is it just random?

**Comment by parkerlreed** [comment]
Maybe it was accidental? The README header was just updated two weeks ago changing it to Tango

https://github.com/yume-chan/ya-webadb/commit/ffb0be2ad15aa449e8deaf62d4496108bef59b3e

**Reply by CorruptionVR** [reply]
oh, ok

**Comment by yume-chan** [comment, marked as answer]
As the project grows, I decided to give it a proper name. Tango was suggested by ChatGPT. It doesn't have any particular meaning. Maybe T for TypeScript, an for Android, and go, for go.

**Reply by CorruptionVR** [reply]
oh ok, that makes sense
